stage7/01-adi-kernel-dts/00-run.sh

```diff
@@ -34,6 +34,16 @@ build_linux $DEFCONFIG_PI3 $KERNEL_IMG_PI3
 build_linux $DEFCONFIG_PI4 $KERNEL_IMG_PI4
 
 cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/overlays/*.dtb* $STAGE_WORK_DIR/rootfs/boot/overlays
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-b.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-b-plus.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-cm.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-zero.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2708-rpi-zero-w.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2709-rpi-2-b.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2710-rpi-3-b.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2710-rpi-3-b-plus.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2710-rpi-cm3.dtb $STAGE_WORK_DIR/rootfs/boot
+cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm2711-rpi-4-b.dtb $STAGE_WORK_DIR/rootfs/boot
```

Comment:
would it be worth to do a wild-card rule?
`cp -f $WORK_DIR/$LINUX_DIR/arch/$ARCH/boot/dts/bcm27*.dtb $STAGE_WORK_DIR/rootfs/boot`
?
for RPi5 or what comes next this needs constant maintenance